lms/djangoapps/bulk_email/models.py

```diff
@@ -168,16 +197,19 @@ def create(
 
         new_targets = []
         for target in targets:
+            # split target, to handle cohort:cohort_name
+            target_split = target.split(':', 1)
             # Ensure our desired target exists
-            if target not in EMAIL_TARGETS:
+            if target_split[0] not in EMAIL_TARGETS:
```

Comment:
I added this, but since it touches the model field, django tries to generate a migration for the change. Given that the list in question is 4 elements, I think suffering the O(n) vs O(1) slowdown is less burdensome than cluttering up the migrations for this djangoapp. Do you agree?